### PR TITLE
[Compute_Disk] - Add beta multi-writer support

### PR DESCRIPTION
**Pull request by ScottSuarez**
Add multi-writer support for Compute Disk. Resolves [6953](https://github.com/hashicorp/terraform-provider-google/issues/6953)

Note: When creating a disk with multiwriter in test scenario I encountered the following error.
Screen Shot 2020-08-25 at 5 42 53 PM

Any advice @danawillow , @c2thorn 

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: added field `multiwriter` to resource `disk` (beta)
```

### CODE REVIEW

**Comment by ScottSuarez** [comment]
After investigating it seems this feature is still in alpha and is whitelist only. 

Will wait to add support until it is released to beta.

**Comment by gdsotirov** [comment]
@ScottSuarez Isn't this feature already in beta? E.g. see https://cloud.google.com/sdk/gcloud/reference/beta/compute/disks/create#--replica-zones

**Comment by ScottSuarez** [comment]
@gdsotirov Hey! Thanks for checking up on this. Last time I looked into it the feature was whitelist only. It's possible that now it's been released to beta. I'll take another look and see if we can support this at this time

**Comment by ScottSuarez** [comment]
@gdsotirov Hey so I've looked into it again its it's still not beta, alpha only. Looks to release at the end of this month pending any issue. Will continue to track on my end.

**Comment by ScottSuarez** [comment]
update.. still waiting for release ...

**Comment by samuelgeek** [comment]
I'm waiting...

**Comment by ScottSuarez** [comment]
@samuelgeek 
https://b.corp.google.com/issues/164841448

**Comment by ScottSuarez** [comment]
* updated with the external link

**Comment by ScottSuarez** [comment]
The issue has now been closed and this feature is now available. I will work to get this in now. 

Thanks for your patience. @samuelgeek @gdsotirov

File: third_party/terraform/tests/resource_compute_instance_test.go.erb [hunk 1]
```diff
@@ -2023,6 +2023,26 @@ func TestComputeInstance_networkIPCustomizedDiff(t *testing.T) {
 	}
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeInstance_multiWriter(t *testing.T) {
```
Comment (rileykarson) [inline third_party/terraform/tests/resource_compute_instance_test.go.erb:2027]: This test should be in Disk because it's primarily a disk test, right?

**Review by rileykarson — APPROVED** [review]
nit: Mind reformatting the changelog? I recognise it's been a while since you wrote it 🙂

File: third_party/terraform/tests/resource_compute_instance_test.go.erb (hunk 1 above)
Comment (ScottSuarez) [reply third_party/terraform/tests/resource_compute_instance_test.go.erb]: yep yep :)

**Pull request merged**